src/string.js

```diff
@@ -0,0 +1,154 @@
+const StringExtension = {
+  /**
+   * Returns true if the string contains vowels.
+   * @returns {Boolean}
+   */
+  hasVowels() {
+    return /[aeiou]/i.test(this);
+  },
+
+   /**
+   * Returns the String in question but with all characters in upper cases as applicable.
+   * @returns {String}
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g,
+      char => String.fromCharCode(char.charCodeAt(0) - 32));
+  },
+
+  /**
+  * Returns the String passed as lower cases
+  * @returns {String}
+  */
+  toLower() {
+    return this.replace(/[A-Z]/g,
+      char => String.fromCharCode(char.charCodeAt(0) + 32));
+  },
+
+  /**
+   * converts strings to title case
+   * @return {String} string that has been converted to title case
+   */
+  ucFirst() {
+    return this.replace(/^(.)/, this[0].toUpper());
+  },
+
+  /**
+   * Checks if the string is a question
+   * @return {Boolean}
+   */
+  isQuestion() {
+    return /\?$/.test(this.trim());
+  },
+
+  /**
+   * checks for words in a string
+   * @return {Array}
+   */
+  words() {
+    return this.match(/\w+/g);
+  },
+
+  /**
+   * Returns the number of words in the string
+   * @returns {Number}
+   */
+  wordCount() {
+    return this.words().length;
+  },
+
+  /**
+   * Returns a currency representation of the String
+   * @returns {String}
+   */
+  toCurrency() {
+    let [number, decimal] = this.split('.');
+    number = number.replace(/,/g, '');
+    if (!decimal) {
+      decimal = '00';
+    }
+    decimal = decimal.substr(0, 2);
+    const result = number.replace(/./g, (char, index, arr) =>
+      (index && (arr.length - index) % 3 === 0 ? `,${char}` : char));
+    return `${result}.${decimal}`;
+  },
+
+  /**
+   * Returns a number representation of the Currency String
+   * @returns {String}
+   */
+  fromCurrency() {
+    return parseFloat(this.replace(/,/g, ''), 10);
```

Comment:
Thanks for the feedback but according to the specification, it is said that the method should return a data type of Number.